src/sass/components/_header.scss

```diff
@@ -222,3 +222,108 @@ header {
     }
   }
 }
+
+// media queries for mobile
```

Comment:
komentarz jest zbędny, to wynika z samej konstrukcji MQ

Reply:
Komentarz usunięty. Zapisałem na potrzeby rozplanowania, zapomniałem usunąć. Już jest ok!